gempython/tools/optohybrid_user_functions_uhal.py

```diff
@@ -122,149 +121,117 @@ def getConnectedVFATsMask(device,gtx=0,debug=False):
+        # reg_list = []
 
-        #CRC counters
-        for vfat in range(24):
-            # reg_list.append("%s.CRC.VALID.VFAT%d"%(    baseNode, vfat))
-            # reg_list.append("%s.CRC.INCORRECT.VFAT%d"%(baseNode, vfat))
-            counters["CRC"]["VALID"]["VFAT%d"%vfat]     = readRegister(device,"%s.CRC.VALID.VFAT%d"%(    baseNode, vfat))
-            counters["CRC"]["INCORRECT"]["VFAT%d"%vfat] = readRegister(device,"%s.CRC.INCORRECT.VFAT%d"%(baseNode, vfat))
+        # #CRC counters
+        # for vfat in range(24):
+        #     # reg_list.append("%s.CRC.VALID.VFAT%d"%(    baseNode, vfat))
+        #     # reg_list.append("%s.CRC.INCORRECT.VFAT%d"%(baseNode, vfat))
+        #     counters["CRC"]["VALID"]["VFAT%d"%vfat]     = readRegister(device,"%s.CRC.VALID.VFAT%d"%(    baseNode, vfat))
+        #     counters["CRC"]["INCORRECT"]["VFAT%d"%vfat] = readRegister(device,"%s.CRC.INCORRECT.VFAT%d"%(baseNode, vfat))
 
-        #T1 counters
-        for t1src in ["GTX_TTC", "GBT_TTC", "INTERNAL","EXTERNAL","LOOPBACK","SENT"]:
-            for t1 in ["L1A", "CalPulse","Resync","BC0"]:
-                # reg_list.append("%s.T1.%s.%s"%(baseNode, t1src, t1))
-                counters["T1"][t1src][t1] = readRegister(device,"%s.T1.%s.%s"%(baseNode, t1src, t1))
+        # #T1 counters
+        # for t1src in ["GTX_TTC", "GBT_TTC", "INTERNAL","EXTERNAL","LOOPBACK","SENT"]:
+        #     for t1 in ["L1A", "CalPulse","Resync","BC0"]:
+        #         # reg_list.append("%s.T1.%s.%s"%(baseNode, t1src, t1))
+        #         counters["T1"][t1src][t1] = readRegister(device,"%s.T1.%s.%s"%(baseNode, t1src, t1))
 
-        # reg_list.append("%s.GTX.TRK_ERR"%(baseNode))
-        # reg_list.append("%s.GTX.TRG_ERR"%(baseNode))
-        # reg_list.append("%s.GTX.DATA_Packets"%(baseNode))
-        counters["GTX"]["TRK_ERR"]      = readRegister(device,"%s.GTX_LINK.TRK_ERR"%(baseNode))
-        counters["GTX"]["DATA_Packets"] = readRegister(device,"%s.GTX_LINK.DATA_Packets"%(baseNode))
-        counters["GBT"]["TRK_ERR"]      = readRegister(device,"%s.GBT_LINK.TRK_ERR"%(baseNode))
-        counters["GBT"]["DATA_Packets"] = readRegister(device,"%s.GBT_LINK.DATA_Packets"%(baseNode))
+        # # reg_list.append("%s.GTX.TRK_ERR"%(baseNode))
+        # # reg_list.append("%s.GTX.TRG_ERR"%(baseNode))
+        # # reg_list.append("%s.GTX.DATA_Packets"%(baseNode))
+        # counters["GTX"]["TRK_ERR"]      = readRegister(device,"%s.GTX_LINK.TRK_ERR"%(baseNode))
+        # counters["GTX"]["DATA_Packets"] = readRegister(device,"%s.GTX_LINK.DATA_Packets"%(baseNode))
+        # counters["GBT"]["TRK_ERR"]      = readRegister(device,"%s.GBT_LINK.TRK_ERR"%(baseNode))
+        # counters["GBT"]["DATA_Packets"] = readRegister(device,"%s.GBT_LINK.DATA_Packets"%(baseNode))
 
-        # reg_vals = readRegisterList(device,reg_list,debug)
+        # # reg_vals = readRegisterList(device,reg_list,debug)
```

Comment:
Probably could just gut these lines entirely no?

Reply:
support this